### DISCUSSION TITLE
state와 props의 차이점은 무엇인가요?

### DISCUSSION THREAD

**Discussion by wonsss** [woowacourse-study/2022-woowahan-bansanghwe#39] [React]
Component [State](https://reactjs.org/docs/faq-state.html) 는 component의 life cycle 동안 변경될 수 있는 정보를 담고 있는 객체입니다.
[Props](https://reactjs.org/docs/components-and-props.html) 는 HTML 태그 속성과 유사한 규칙을 사용하여 React component 에 전달되는 값을 포함하는 단일 값 또는 객체입니다. 부모 component 에서 자식 component 로 전달되는 데이터 입니다.

props와 state는 모두 JavaScript 객체입니다. 두가지 다 렌더링 결과에 영향을 주는 정보를 가지고 있지만, component 와 관련된 기능면에서 차이가 있습니다.
props는 함수 매개변수와 같이 component 요소로 전달됩니다. state 는 component 안에서 관리되고 사용할 변수 선언과 비슷합니다.

**Comment by InKyoJeong** [comment]
state와 props,,정말 기본적인 개념같지만 막상 갑자기 뭐냐 하면 바로 설명하기 어려운거같아요.
